src/utils/oldProvider/oldHttpProvider.ts

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
I don't think we should copy-paste web3.js code into our library. If it's used only for testing, why don't we add web3 as a dev dependency?

Reply:
@bogdan we can bring several versions of the same dependency using package aliases
https://stackoverflow.com/a/56495651/4284929
I haven't tested it myself, but I believe it should work (we can try at least)

0.x being not split into subpackages shouldn't be a problem, as long as it's a dev dependency

Reply:
Well, lets try that if it could be easy enough. I don't believe in the flexibility of package.json because disappointed me a lot of times. In this particular case,  there might problems with a dependencies of the dependencies.

Reply:
It worked!